packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2417,13 +2447,20 @@ is required to delete the user.
 | id        | `id` of user                                                                          | URL        | String   | TRUE     |         |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    | FALSE   |
 
+> Note: You can also perform user deletion asynchronously. To invoke asynchronous user deletion, pass an HTTP header
```

Comment:
the word Note and the colon are bold. I don't think that came across in my comments. Sorry. So > and then two asterisks Note: and two asterisks. Sorry about that.

Reply:
Updated.